src/EnergyPlus/SingleDuct.cc

```diff
@@ -1791,8 +1791,11 @@ namespace SingleDuct {
 		//Do a check and make sure that the max and min available(control) flow is
 		//  between the physical max and min while operating.
 		SysInlet( SysNum ).AirMassFlowRateMaxAvail = min( Sys( SysNum ).AirMassFlowRateMax, Node( InletNode ).MassFlowRateMaxAvail );
-		SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
-
+		if ( Sys( SysNum ).SysType_Num == SingleDuctConstVolReheat ) { // AirTerminal:SingleDuct:ConstantVolume:Reheat
+			SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
+		} else { // #6173
+			SysInlet( SysNum ).AirMassFlowRateMinAvail = min( Node( OutletNode ).MassFlowRateMin, SysInlet( SysNum ).AirMassFlowRateMaxAvail );
+		}
```

Comment:
I realize that this change works in this case. But I also think the MinAvail is needed to ensure that other components provide a flow rate within constraints imposed elsewhere in the simulation. So, in order for this to move forward, the reason that MinAvail is higher than required here needs to be identified. Otherwise I am not willing to accept this change. Other reviewers might be willing to accept this change.